port/unix/omrsysinfo.c

```diff
@@ -3768,8 +3796,11 @@ uint64_t
 omrsysinfo_cgroup_are_subsystems_available(struct OMRPortLibrary *portLibrary, uint64_t subsystemFlags)
 {
 #if defined(LINUX) && !defined(OMRZTPF)
+	Trc_PRT_sysinfo_cgroup_are_subsystems_available_Entry(subsystemFlags);
 	uint64_t available = portLibrary->sysinfo_cgroup_get_available_subsystems(portLibrary);
-	return (available & subsystemFlags);
+	uint64_t rc = available & subsystemFlags;
+	Trc_PRT_sysinfo_cgroup_are_subsystems_available_Exit(rc);
```

Comment:
This code isn't c89 - the variables would need to be declared before the tracepoint.